client.go

```diff
@@ -117,6 +119,71 @@ func (c *client) GetBug(id int) (*Bug, error) {
 	return bugs[0], nil
 }
 
+// GetBugComments retrieves the comments of a Bug from the server
+// https://bugzilla.readthedocs.io/en/latest/api/core/v1/comment.html#get-comments
+func (c *client) GetBugComments(id int, values *url.Values) ([]Comment, error) {
```

Comment:
I don't think I like the idea of exposing url.Values as the public API.  What are we doing with this? Is this a leaky way to set comment_id or new_since?

Reply:
new_since. Am happy to add optional option structs.

Reply:
fixed (without option structs)